### PR TITLE
Add to README

### PR DESCRIPTION
**Pull request by JelleZijlstra**

### CODE REVIEW

File: src/typing_extensions.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 import typing
 
 
-# Please keep __all__ alphabetized within each category.
```
Comment (JelleZijlstra) [inline src/typing_extensions.py:11]: They're currently not alphabetized and I don't care.

File: README.md [hunk 2]
```diff
@@ -50,13 +52,13 @@ This module currently contains the following:
...
   - `Self` (see PEP 673)
-  - `TypeVarTuple` (see PEP 646)
+  - `TypeVarTuple` (see PEP 646; the `typing_extensions` version supports the `default=` argument from PEP 697)
   - `Unpack` (see PEP 646)
 
 - In `typing` since Python 3.10
 
   - `Concatenate` (see PEP 612)
-  - `ParamSpec` (see PEP 612)
+  - `ParamSpec` (see PEP 612; the `typing_extensions` version supports the `default=` argument from PEP 697)
```
Comment (AlexWaygood) [inline README.md:61]:
```suggestion
  - `ParamSpec` (see PEP 612; the `typing_extensions` version supports the `default=` argument from PEP 696)
```

File: README.md [hunk 3]
```diff
@@ -105,6 +106,13 @@ This module currently contains the following:
   - `TYPE_CHECKING`
   - `get_type_hints`
 
+- The following have always been present in `typing`, but the `typing_extensions` versions provide
+  additional features:
+
+  - `Any` (supports inheritance since Python 3.11)
+  - `NamedTuple` (supports inheritance from `Generic` since Python 3.11)
+  - `TypeVar` (see PEPs 695 and 697)
```
Comment (AlexWaygood) [inline README.md:114]:
```suggestion
  - `TypeVar` (see PEPs 695 and 696)
```

File: README.md [hunk 4]
```diff
@@ -125,6 +133,11 @@ Certain objects were changed after they were added to `typing`, and
   `@typing_extensions.overload`.
 - `NamedTuple` was changed in Python 3.11 to allow for multiple inheritance
   with `typing.Generic`.
+- Since Python 3.11, it has been possible to inherit from `Any` at
+  runtime. `typing_extensions.Any` also provides this capability.
+- `TypeVar` gains two additional parameters, `default=` and `infer_variance=`,
+  in the draft PEPs 695 and 697, which are being considered for inclusion
```
Comment (AlexWaygood) [inline README.md:139]:
```suggestion
  in the draft PEPs 695 and 696, which are being considered for inclusion
```

File: README.md [hunk 5]
```diff
@@ -105,6 +106,13 @@ This module currently contains the following:
   - `TYPE_CHECKING`
   - `get_type_hints`
 
+- The following have always been present in `typing`, but the `typing_extensions` versions provide
+  additional features:
+
+  - `Any` (supports inheritance since Python 3.11)
+  - `NamedTuple` (supports inheritance from `Generic` since Python 3.11)
```
Comment (AlexWaygood) [inline README.md:113]:
```suggestion
  - `NamedTuple` (supports multiple inheritance with `Generic` since Python 3.11)
```

**Review by AlexWaygood — APPROVED** [review]

**Pull request merged**